### PR TITLE
New package: python3-pillow-simd-5.3.0.post1

### PR DESCRIPTION
**Pull request by mustaqimM**

### CODE REVIEW

**Comment by Hoshpak** [comment]
Unless the package detects the capabilities of the target platform, I would argue that it should be `x86_64*` only since SSE4 was introduced with Core2 processors and there are no x86_32 processors that support it.

File: srcpkgs/pillow-simd/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'pillow-simd'
+pkgname=pillow-simd
```
Comment (Hoshpak) [inline srcpkgs/pillow-simd/template]: please rename the package `python3-pillow-simd` to fit into the naming scheme of python packages.

File: srcpkgs/pillow-simd/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+version=5.3.0.post1
+revision=1
+archs="x86_64*"
+build_style=python3-module
+pycompile_module="PIL"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel libjpeg-turbo-devel libopenjpeg2-devel
+ zlib-devel tiff-devel freetype-devel lcms2-devel libwebp-devel libimagequant-devel freetype-devel harfbuzz-devel fribidi-devel libraqm-devel"
```
Comment (Hoshpak) [inline srcpkgs/pillow-simd/template]: this line is too long, please add line breaks at aproximately 80 characters.

Comment (mustaqimM) [reply srcpkgs/pillow-simd/template]: okay, I've made the necessary changes.

File: srcpkgs/pillow-simd/template (hunk 1 above)
Comment (Hoshpak) [reply srcpkgs/pillow-simd/template]: You also need to rename the directory and change the comment line to the new package name

Comment (DirectorX) [reply srcpkgs/pillow-simd/template]: And the commit message

Comment (mustaqimM) [reply srcpkgs/pillow-simd/template]: It seems to be resolved.

File: srcpkgs/python3-pillow-simd/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'pillow-simd'
```
Comment (Hoshpak) [inline srcpkgs/python3-pillow-simd/template]: This line should contain the same name as `pkgname`

File: srcpkgs/python3-pillow-simd/template [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel libjpeg-turbo-devel libopenjpeg2-devel
+ zlib-devel tiff-devel freetype-devel lcms2-devel libwebp-devel
+ libimagequant-devel freetype-devel harfbuzz-devel fribidi-devel libraqm-devel"
+short_desc="Python3 Imaging Library (PIL) fork (SIMD)"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="custom: PIL"
+homepage="https://python-pillow.org/pillow-perf/"
+changelog="https://github.com/uploadcare/pillow-simd/compare/v5.2.0.post1...v5.3.0.post1"
+distfiles="https://github.com/uploadcare/${pkgname}/archive/v${version}.tar.gz"
```
Comment (Hoshpak) [inline srcpkgs/python3-pillow-simd/template]: This is broken since the url does not match `pkgame` any more.

Comment (mustaqimM) [reply srcpkgs/python3-pillow-simd/template]: okay, I've resolved the last of the issues

**Comment by Johnnynator** [comment]
`=> ERROR: python3-pillow-simd-5.3.0.post1_1: cannot access wrksrc directory [/builddir/python3-pillow-simd-5.3.0.post1]`

You need to set `wrksrc="pillow-simd-${version}"`

**Pull request merged**